README.md

```diff
@@ -152,6 +153,16 @@ You can also add your own env files other than `dev` and `prod` by doing the fol
 - add `{ NAME: 'src/environments/environment.NAME.ts' }` to the the `apps[0].environments` object in `angular-cli.json` 
 - use them by using the `--env=NAME` flag on the build/serve commands.
 
+### Base tag handling in index.html
+
+You can modify base tag (`<base href="/">`) in your index.html by using `--base-href your-url` option. It's useful when building or serving for different environments.
+
+```bash
+# Sets base tag href to /myUrl/ in your index.html
+ng build --base-href /myUrl/
+ng serve --base-href /myUrl/
```

Comment:
Heh. You're right! The given example will not work as expected. 
I was just thinking about the opposite case: when using production value in index.html and you need to override it back to "/" when developing. But seems that would be the only working case. So I will remove it form `serve` and rebase master.